packages/fhir-converter/src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -1058,4 +1060,111 @@ module.exports.external = [
       return PERSONAL_RELATIONSHIP_TYPE_CODE;
     },
   },
+  {
+    name: "decodeHtmlEntities",
+    description: "Decodes html strings",
+    func: function (str) {
+      if (str === undefined || str === null) {
+        return "";
+      }
+      const result = he.decode(str);
+      return result;
+    },
+  },
+  {
+    name: "removeNonNumeric",
+    description: "Removes all non-numeric characters from a string, keeping decimal points",
+    func: function (str) {
+      if (str === undefined || str === null) {
+        return "";
+      }
+      return str.replace(/[^0-9.]/g, '');
+    },
+  },
+  {
+    name: "convertFeetAndInchesToCm",
+    description: "Checks if a string is in the format 'number ft number in' and if so, converts the feet and inches to centimeters",
```

Comment:
Why did I do this? We get values for height observation often like this, but there is no way to convey 5 ft 2 as a decimal, which is a requirement so HAPI doesn't BARF. Metric system lets us use decimals so I took this approach

Reply:
That one will get handled by `extractNumberAndUnit`
😄